ci-operator/config/ViaQ/vector/ViaQ-vector-v0.46.1-rh.yaml

```diff
@@ -0,0 +1,179 @@
+base_images:
+  builder:
+    name: ubi
+    namespace: ocp
+    tag: "9"
+  cluster-logging-operator:
+    name: 6.y
+    namespace: logging
+    tag: cluster-logging-operator
+  cluster-logging-operator-e2e:
+    name: 6.y
+    namespace: logging
+    tag: cluster-logging-operator-e2e
+  cluster-logging-operator-registry:
+    name: 6.y
+    namespace: logging
+    tag: cluster-logging-operator-registry
+  elasticsearch-operator:
+    name: "5.8"
+    namespace: logging
+    tag: elasticsearch-operator
+  elasticsearch-operator-registry:
+    name: "5.8"
+    namespace: logging
+    tag: elasticsearch-operator-registry
+  elasticsearch-proxy:
+    name: 5.y
+    namespace: logging
+    tag: elasticsearch-proxy
+  log-file-metric-exporter:
+    name: 6.y
+    namespace: logging
+    tag: log-file-metric-exporter
+  logging-elasticsearch6:
+    name: 5.y
+    namespace: logging
+    tag: logging-elasticsearch6
+  logging-eventrouter:
+    name: 5.y
+    namespace: logging
+    tag: logging-eventrouter
+  oauth-proxy:
+    name: "4.14"
+    namespace: ocp
+    tag: oauth-proxy
+  ubi9-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
```

Comment:
I think this is for core openshift but I think you could bump this to 1.23.  Maybe try grep'ing the repo to see if its anywhere else